src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -681,7 +681,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Dim parameter As IParameterSymbol = boundParameterEqualsValue.Parameter
             Dim value As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundParameterEqualsValue.Value))
             Dim kind As OperationKind = OperationKind.ParameterInitializerAtDeclaration
-            Dim isInvalid As Boolean = DirectCast(boundParameterEqualsValue.Value, IOperation).IsInvalid
+            Dim isInvalid As Boolean = value.Value.IsInvalid
```

Comment:
Could we simply check boundParameterEqualsValue.Value.HasErrors instead?

---
In reply to: [116079177](https://github.com/dotnet/roslyn/pull/19406#discussion_r116079177) [](ancestors = 116079177)